ToDoApp/ToDoApp/ToDoListView.swift

```diff
@@ -13,51 +13,41 @@ struct ToDoSection: Identifiable {
     let toDoItems: [TaskModel]
 }
 struct ToDoListView: View {
-    static let items: [ToDoSection] = [
-            .init(title: "Today’s To-Do List", toDoItems: [
-                .init(title: "Cleaning", deadline: Date(), status: false),
-                .init(title: "Cooking", deadline: Date(), status: true),
-                .init(title: "Painting", deadline: Date(), status: true),
-                .init(title: "Learning", deadline: Date(), status: true),
-                .init(title: "Midterm", deadline: Date(), status: true),
-                .init(title: "Laundry", deadline: Date(), status: false),
-                .init(title: "Clean car", deadline: Date(), status: true),
-                .init(title: "Workout", deadline: Date(), status: true),
-                .init(title: "Cooking", deadline: Date(), status: false)
-                
-            ]),
-            .init(title: "All To-Do List", toDoItems: [
-                .init(title: "Laundry", deadline: Date(), status: false),
-                .init(title: "Cleaning", deadline: Date(), status: true),
-                .init(title: "Final", deadline: Date(), status: false),
-                .init(title: "Grocery", deadline: Date(), status: true),
-                .init(title: "Dancing", deadline: Date(), status: false),
-                .init(title: "Workout", deadline: Date(), status: false),
-                .init(title: "Laundry", deadline: Date(), status: false),
-                .init(title: "Drawing", deadline: Date(), status: false),
-                .init(title: "Painting", deadline: Date(), status: true)
-            ])
-        ]
+    @StateObject var viewModel = ToDoListViewModel()
     var body: some View {
         NavigationStack{
-            List {
-                ForEach(ToDoListView.items) { tasks in
-                    HStack(spacing: 6) {
-                        Text(tasks.title)
-                            .foregroundColor(Color(.caption))
-                            .fontWeight(.medium)
-                        Text("\(tasks.toDoItems.count) tasks left")
-                            .foregroundColor(Color(.secondaryText))
-                            
-                    }
-                    .font(.caption)
-                    ForEach(tasks.toDoItems) { todo in
-                        TaskCell(todo: todo)
+            Group {
+                switch viewModel.state {
+                    case .idle:
+                        EmptyView()
```

Comment:
the alignment is broken, you can use control + i shortcut to format a file in Xcode